task/kamel-run/0.1/kamel-run.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: kamel-run
```

Comment:
shouldn't this be `camelK` or something like that, not very sure 😅

Reply:
No, this is a bit more opinionated than a generic Camel-K client. The idea is that the user is able only to perform a running operation (hence, `kamel-run`). In the future we may think to extend to other operations, but we want to have a very specific behavior and not leave a generic CLI open to the user.